pkg/cmd/server/bootstrappolicy/dead.go

```diff
@@ -21,7 +21,13 @@ func addDeadClusterRole(name string) {
 
 	deadClusterRoles = append(deadClusterRoles,
 		authorizationapi.ClusterRole{
-			ObjectMeta: metav1.ObjectMeta{Name: name},
+			ObjectMeta: metav1.ObjectMeta{
+				Name: name,
+				Annotations: map[string]string{
+					// typical users should not see dead cluster roles
```

Comment:
No one should ever see this really since they are just placeholders for reconciliation.